### PR TITLE
put conditional locks on nbc bump

### PR DESCRIPTION
**Pull request by sinkingsugar**

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -207,6 +207,7 @@ jobs:
           nimble test
 
   bumpNBC-stable:
+    if: github.ref == 'refs/heads/master'
```
Comment (dryajov) [inline .github/workflows/ci.yml:210]: We should change to `master` to `stable`.

Comment (jakubgs) [reply .github/workflows/ci.yml:210]: Agreed.

Comment (sinkingsugar) [reply .github/workflows/ci.yml:210]: I propose we keep master for backward compatibility and just leave it as it is tho, does that work for you @dryajov ?
And of course set stable as default.

Comment (sinkingsugar) [reply .github/workflows/ci.yml:210]: Altho dunno, ref might be enough, someone pulling master might be better off with error message to know they have to change.

**Pull request merged**